src/Toolkit/Toolkit/UI/Controls/FeatureForm/GroupFormElementView.Maui.cs

```diff
@@ -0,0 +1,271 @@
+﻿// /*******************************************************************************
+//  * Copyright 2012-2018 Esri
+//  *
+//  *  Licensed under the Apache License, Version 2.0 (the "License");
+//  *  you may not use this file except in compliance with the License.
+//  *  You may obtain a copy of the License at
+//  *
+//  *  http://www.apache.org/licenses/LICENSE-2.0
+//  *
+//  *   Unless required by applicable law or agreed to in writing, software
+//  *   distributed under the License is distributed on an "AS IS" BASIS,
+//  *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  *   See the License for the specific language governing permissions and
+//  *   limitations under the License.
+//  ******************************************************************************/
+
+
+#if MAUI
+using System.ComponentModel;
+using System.Diagnostics.CodeAnalysis;
+using Microsoft.Maui.Controls.Internals;
+using Esri.ArcGISRuntime.Mapping.FeatureForms;
+using Esri.ArcGISRuntime.Toolkit.Internal;
+using System.Globalization;
+using Microsoft.Maui.Controls.Shapes;
+
+namespace Esri.ArcGISRuntime.Toolkit.Maui.Primitives
+{
+    /// <summary>
+    /// Groups a set of <see cref="FieldFormElementView"/> views in a collapsible group.
+    /// </summary>
```

Comment:
Move the summary to the cross-platform part (GroupFormElementView.cs) otherwise WPF doc gets no summary at all.

Reply:
There is no WPF version of this one.

Reply:
Oops. I swear I saw a "missing summary" warning but I think it was addressed mid-review by 51f187cee768d5c1c17c8f521d9f23d3641e9bff